src/Microsoft.DotNet.ProjectModel/Compilation/LibraryExporter.cs

```diff
@@ -200,7 +200,18 @@ private LibraryExport ExportProject(ProjectDescription project)
             else if (project.Project.Files.SourceFiles.Any())
             {
                 var outputPaths = project.GetOutputPaths(_buildBasePath, _solutionRootPath, _configuration, _runtime);
-                var files = outputPaths.CompilationFiles;
+                CompilationOutputFiles files;
+
+                if (project == _rootProject &&
```

Comment:
This will be changed it in next pr with asset refactoring anyway, so I'd prefer just merging this one.